src/languages/clle/getmbr.ts

```diff
@@ -0,0 +1,49 @@
+import Instance from "../../api/Instance";
+
+export async function initGetMbr(instance: Instance) {
+  const connection = instance.getConnection();
+  const config = instance.getConfig();
+
+  // Check if the remote library list tool is installed
+  if (connection && config && !connection.remoteFeatures[`GETMBR.PGM`]) {
+    // Time to install our new library list fetcher program
+    const content = instance.getContent()!;
+  
+    const tempLib = config.tempLibrary;
+  
+    try {
+      await connection.remoteCommand(`CRTSRCPF ${tempLib}/QTOOLS`, undefined)
+    } catch (e) {
+      //It may exist already so we just ignore the error
+    }
+  
+    await content.uploadMemberContent(undefined, tempLib, `QTOOLS`, `GETMBR`, getSource());
+    await connection.remoteCommand(
```

Comment:
`remoteCommand` is deprecated and `runCommand` should be used instead:

```suggestion
    await connection.runCommand(
```